src/pages/SearchPage.js

```diff
@@ -175,6 +198,7 @@ class SearchPage extends Component {
                             onCloseButtonPress={() => Navigation.dismissModal(true)}
                         />
                         <View style={[styles.flex1, styles.w100, styles.pRelative]}>
+                            {this.props.reportsServerSearchDataLoading && <Text>Loading reports from server!</Text>}
```

Comment:
Maybe we can just add more reports when they show up and not do any UI polish yet to indicate a loading state.

Reply:
Removed